app/controllers/devise_sessions_controller.rb

```diff
@@ -88,11 +117,11 @@ def verify_signed_out_user; end
 
   def check_login_state
     @login_state_id = session[:login_state_id]
-    redirect_to new_user_session_path unless login_state
+    redirect_to user_session_path unless login_state
```

Comment:
Is there a `user_session_path` any more?

Reply:
Have changed to `new_user_session_path`